documentation/asciidoc/topics/con_anti_affinity.adoc

```diff
@@ -0,0 +1,31 @@
+[id='anti_affinity-{context}']
+= Anti-Affinity Strategies
+Each {brandname} node in a cluster is a pod that runs on an {k8s} node in a cluster.
+Each {openshift} node runs on a physical host system.
+Anti-affinity works by distributing {brandname} nodes across {k8s} nodes, ensuring that your {brandname} clusters remain available even if hardware failures occur.
+
+[discrete]
+== Strategies
+
+{ispn_operator} offers two anti-affinity strategies:
+
+Hostname:: {brandname} replica pods are scheduled on different {k8s} nodes.
+Zone:: {brandname} replica pods are scheduled across multiple zones.
+
+[discrete]
+== Fault tolerance
+
+`spec.replicas`:: Defines the number of nodes in each {brandname} cluster.
```

Comment:
This should be: "Defines the number of pods in each {brandname} cluster."

Reply:
nice catch. fixed.